examples/callbacks/src/lib.rs

```diff
@@ -2,25 +2,33 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-pub trait OnCallAnswered {
-    fn hello(&self) -> String;
-    fn busy(&self);
-    fn text_received(&self, text: String);
+#[derive(Debug, thiserror::Error)]
+pub enum TelephoneError {
+    #[error("Busy")]
+    Busy,
+    #[error("InternalTelephoneError")]
+    InternalTelephoneError,
+}
+
+impl From<uniffi::UnexpectedUniFFICallbackError> for TelephoneError {
```

Comment:
I think a comment here - even just to point at the doc changes you made - might help people unfamiliar with uniffi and checking out the examples.